Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkEditor.cs

```diff
@@ -84,10 +84,17 @@ public void LinkAll()
 
         public static void LinkAll(AudioLink target)
         {
+#if UNITY_6000_0_OR_NEWER
+#if UDONSHARP
+            MonoBehaviour[] allBehaviours = FindObjectsByType<UdonSharpBehaviour>(FindObjectsInactive.Include, FindObjectsSortMode.InstanceID);
```

Comment:
Is this intended to be MonoBehaviour and not UdonSharpBehaviour?

Reply:
Nope, this was a copy-paste typo. It should've been UdonSharpBehaviour[]. float3's changes handle it better anyways.